### PR TITLE
chore: added ava for testing

### LINKED ISSUE #59: Add AVA for testing
## Description

I need to write some tests for a fix and we currently do not have testing in place for the project.

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Added AVA for tests.

Lemurs hopping over one lemu

## Related issues and documents

Closes #59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,21 @@
...
+  },
+  "ava": {
+    "extensions": {
+      "ts": "module"
+    },
+    "nodeArguments": [
+      "--loader=esbuild-node-loader"
+    ],
+    "files": [
+      "tests/**/*.test.ts"
```
Comment (nickytonline) [inline package.json:51]: I prefer colocating tests, but I notice we typically have a tests folder for tests so went with the same format.

File: tests/fake.test.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import test from 'ava'
```
Comment (nickytonline) [inline tests/fake.test.ts:1]: I'll remove this once I add my tests.

File: package.json [hunk 3]
```diff
@@ -34,8 +34,21 @@
...
     "typescript": "^4.6.2"
+  },
+  "ava": {
+    "extensions": {
+      "ts": "module"
+    },
+    "nodeArguments": [
+      "--loader=esbuild-node-loader"
+    ],
+    "files": [
```
Comment (nickytonline) [inline package.json:50]: I prefer colocating tests, but I notice we typically have a tests folder for tests so went with the same format.

File: test/fake.test.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import test from 'ava'
```
Comment (nickytonline) [inline test/fake.test.ts:1]: I'll remove this once I add my tests.

**Review by ericapisani — APPROVED** [review]

**Pull request merged**